application/modules/front/views/collection/hadith_reference.php

```diff
@@ -1,4 +1,7 @@
 <?php
+			use app\modules\front\models\Util;
+			$util = new Util();
```

Comment:
Is it possible to do this in the controller? I would like to move away from computations like this, especially involving `Util` in the view.